src/main/java/frc/team166/robot/subsystems/Drive.java

```diff
@@ -31,16 +33,16 @@
 public class Drive extends Subsystem {
 
     // declare lidar
-    Lidar frontLidar = new Lidar(Port.kOnboard, 0x10);
+    Lidar frontLidar = new Lidar(Port.kOnboard, 0x30, 10);
     //defines the gyro
     AnalogGyro tempestGyro = new AnalogGyro(RobotMap.AnalogInputs.tempestgyro);
     //defines the left motors as motors and combines the left motors into one motor
-    WPI_VictorSPX m_rearleft = new WPI_VictorSPX(RobotMap.CAN.BACK_LEFT);
-    WPI_VictorSPX m_frontleft = new WPI_VictorSPX(RobotMap.CAN.FRONT_LEFT);
+    WPI_TalonSRX m_rearleft = new WPI_TalonSRX(RobotMap.CAN.BACK_LEFT);
```

Comment:
Can we add a TODO comment around all the "Talon" stuff so we remember to change it back before real-robot integration?